src/dbnode/storage/cluster/database.go

```diff
@@ -131,6 +131,10 @@ func (d *clusterDB) Topology() topology.Topology {
 	return d.topo
 }
 
+func (d *clusterDB) TopologyMap() (topology.Map, error) {
+	return d.topo.Get(), nil
```

Comment:
This can still be racy, because two watches can get the update at different times. Instead of this, create a struct field in clusterDB which tracks topology.Map, and is updated based on the value as viewed by the clusterDB.

Reply:
spoke offline, agreed for now that its ok if the bootstrapp process receives a topology snapshot that is *newer* than the one that triggered the bootstrap as long as its not older.

#1013